app/components/markdown/markdown_image/index.tsx

```diff
@@ -81,13 +81,7 @@ const MarkdownImage = ({
     const originalSize = getMarkdownImageSize(isReplyPost, isTablet, sourceSize, metadata, layoutWidth);
     const serverUrl = useServerUrl();
     const galleryIdentifier = `${postId}-${genericFileId}-${location}`;
-    const uri = useMemo(() => {
-        if (source.startsWith('/')) {
-            return serverUrl + source;
-        }
-
-        return source;
-    }, [source, serverUrl]);
+    const uri = source.startsWith('/') ? serverUrl + source : source;
 
     const fileInfo = useMemo(() => {
```

Comment:
Nit: This is memoizing a function. Shouldn't it be useCallback? (I think there is not much difference, just sounds more correct)

Reply:
fileInfo is an object not a function.

Reply:
My bad :sweat_smile: